### PR TITLE
Updates to the second Duality testnet configuration

### PR DESCRIPTION
**Pull request by teddyknox**

### CODE REVIEW

File: replicated-security/duality-testnet-1/README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 
 # `duality` Chain Details
 
-The `duality-second-rehearsal-1` chain will be launched as a persistent chain to test Interchain Security functionality.
+The `duality-testnet-1` chain will be launched as a persistent chain to test Interchain Security functionality.
 
-* **Chain-ID**: `duality-second-rehearsal-1`
+* **Chain-ID**: `duality-testnet-1`
```
Comment (dasanchez) [inline replicated-security/duality-testnet-1/README.md:6]: This doesn't match the chain ID listed in the genesis file.

File: replicated-security/duality-testnet-1/join-rs-duality-testnet-1.sh [hunk 2]
```diff
@@ -12,16 +12,16 @@ set -e
...
 NODE_HOME=~/.duality
-NODE_MONIKER=duality-second-rehearsal-1
+NODE_MONIKER=duality-testnet-1
 SERVICE_NAME=duality
 # ***
 
 CHAIN_BINARY='dualityd'
-CHAIN_ID=duality-second-rehearsal-1
-SEEDS="cebd80328abca02408d1b7199f9b1a835a14aa31@p2p.testnet-1.duality.xyz:26656"
+CHAIN_ID=duality-testnet-1
```
Comment (dasanchez) [inline replicated-security/duality-testnet-1/join-rs-duality-testnet-1.sh:20]: This doesn't match the genesis file chain ID.

File: replicated-security/duality-testnet-1/join-rs-duality-testnet-1-cv.sh [hunk 3]
```diff
@@ -12,16 +12,16 @@ set -ex
...
 NODE_HOME=~/.duality
-NODE_MONIKER=duality-second-rehearsal-1
+NODE_MONIKER=duality-testnet-1
 SERVICE_NAME=cv-duality
 # ***
 
 CHAIN_BINARY='dualityd'
-CHAIN_ID=duality-second-rehearsal-1
-SEEDS="cebd80328abca02408d1b7199f9b1a835a14aa31@p2p.testnet-1.duality.xyz:26656"
+CHAIN_ID=duality-testnet-1
```
Comment (dasanchez) [inline replicated-security/duality-testnet-1/join-rs-duality-testnet-1-cv.sh:20]: This doesn't match the genesis file chain ID

File: replicated-security/duality-testnet-1/proposal-start-duality-testnet-1.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+    "title": "Create the duality chain",
+    "description": "# Launching the duality-testnet-1 Chain\r\n\r\nThis is the proposal to create the first persistent duality chain, \"duality-testnet-1\".\r\n\r\nPlease visit the [cosmos/testnets](https://github.com/cosmos/testnets/tree/master/replicated-security/duality-testnet-1/) repo for additional details.",
+    "chain_id": "duality-testnet-1",
```
Comment (dasanchez) [inline replicated-security/duality-testnet-1/proposal-start-duality-testnet-1.json:4]: This doesn't match the genesis file chain ID

**Review by dasanchez — COMMENTED** [review]
Hi @teddyknox ! I think the chain ID in the genesis file needs to be fixed.

File: replicated-security/duality-testnet-1/join-rs-duality-testnet-1-cv.sh (hunk 3 above)
Comment (teddyknox) [reply replicated-security/duality-testnet-1/join-rs-duality-testnet-1-cv.sh:20]: Done

File: replicated-security/duality-testnet-1/proposal-start-duality-testnet-1.json (hunk 4 above)
Comment (teddyknox) [reply replicated-security/duality-testnet-1/proposal-start-duality-testnet-1.json:4]: Done.

**Review by dasanchez — APPROVED** [review]
Thanks @teddyknox !

**Pull request merged**